Src/FluentAssertions/Equivalency/IValidateChildNodeEquivalency.cs

```diff
@@ -1,9 +1,9 @@
 namespace FluentAssertions.Equivalency;
 
-public interface IEquivalencyValidator
+public interface IValidateChildNodeEquivalency
```

Comment:
Only info, nothing to change.
I know this is one point where we differ, as I don't see this renaming as an improvement.

Reply:
> I know this is one point where we differ, as I don't see this renaming as an improvement.

I know, but I'm trying to apply Interface Seggregation here. It comes up every time as feedback from my students every time I run my TDD/Clean Code workshop ;-)